sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/Data.java

```diff
@@ -1037,7 +1068,7 @@ public void run() {
                                     eTag),
                             null);
                 }
-                if (pendingOperation.isExpired() || PENDING_OPERATION_DELETE_VALUE.equals(pendingOperation.getOperation())) {
+                if (PENDING_OPERATION_DELETE_VALUE.equals(pendingOperation.getOperation())) {
 
                     /* Remove the document if expiration_time has elapsed or it is a delete operation. */
```

Comment:
Comment out of date when removing one of the condition (also why is this change needed?).

Reply:
the pendOperation expiration check will happened in the processPendingOperation now, that's why removed from here :D